src/AppBundle/Entity/Shift.php

```diff
@@ -115,6 +106,11 @@ class Shift
      */
     private $timeLogs;
 
+    /**
+     * @ORM\OneToMany(targetEntity="ShiftFreeLog", mappedBy="shift")
+     */
+    private $shiftfree;
```

Comment:
```suggestion
    private $shiftFreeLogs;
```
pour matcher avec `timeLogs`